### PR TITLE
Added fruit salad

### PR DESCRIPTION
**Pull request by cgphelps**
For issue #1

### CODE REVIEW

**Review by ahardin-rh — APPROVED** [review]
Looks delish

File: appetizer.md [hunk 1]
```diff
@@ -4,6 +4,11 @@
 * Guacamole
 * Breadsticks
 * Chips and salsa
+* Fruit Salad
+ * Blueberries
```
Comment (JasonTestAccount) [inline appetizer.md:8]: were out

**Comment by cgphelps** [comment]
@ahardin-rh Both look good, merged your sliders in!

**Review by JasonTestAccount — CHANGES_REQUESTED** [review]
Check on the blueberries. Did you buy some?

**Pull request merged**